components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/StandaloneJobOrchestratorTests.cs

```diff
@@ -111,6 +111,7 @@ public class StandaloneJobOrchestratorTests : TempFileProviderBase {
             Assert.Equal(endpoint.OpcAuthenticationPassword, null);
             Assert.Equal(endpoint.DataSetWriterId, "DataSetWriterId0");
             Assert.Equal(endpoint.DataSetPublishingInterval, 10000);
+            Assert.Equal(endpoint.Tag, "Tag0");
```

Comment:
Can we do this check in some other test ? This test checks parsing of legacy `pn.json` files which didn't contain `Tag` field.

This comment is related to the ask to remove `Tag` field from `pn_2.5_legacy.json` file.

Reply:
ok